docs/user/openstack/README.md

```diff
@@ -526,84 +526,108 @@ property of the MachineSet.
-
-```sh
-openstack server list
-```
+- The master nodes serve the OpenShift API on port 6443 using TCP.
+- The apps hosted on the worker nodes are served on ports 80, and 443. They are both served using TCP.
 
-These floating IPs can then be used by the load balancer to access the cluster.  An example of HAProxy configuration for port 6443 is below.
+Note: Make sure the instance that your new load balancer is running on has security group rules that allow TCP traffic over these ports.
 
-```txt
-listen <cluster name>-api-6443
-    bind 0.0.0.0:6443
-    mode tcp
-    balance roundrobin
-    server <cluster name>-master-2 <floating ip>:6443 check
-    server <cluster name>-master-0 <floating ip>:6443 check
-    server <cluster name>-master-1 <floating ip>:6443 check
-```
+#### HAProxy Example Load Balancer Config
 
-The other port configurations are identical.
+The following `HAProxy` config file demonstrates a basic configuration for an external load balancer:
 
-The next step is to allow network access from the load balancer network to the master nodes:
-
-```sh
-openstack security group rule create master --remote-ip <load balancer CIDR> --ingress --protocol tcp --dst-port 6443
-openstack security group rule create master --remote-ip <load balancer CIDR> --ingress --protocol tcp --dst-port 443
-openstack security group rule create master --remote-ip <load balancer CIDR> --ingress --protocol tcp --dst-port 80
+```haproxy
+listen <cluster-name>-api-6443
+        bind 0.0.0.0:6443
+        mode tcp
+        balance roundrobin
+        server <cluster-name>-master-0 192.168.0.154:6443 check
+        server <cluster-name>-master-1 192.168.0.15:6443 check
+        server <cluster-name>-master-2 192.168.3.128:6443 check
+listen <cluster-name>-apps-443
+        bind 0.0.0.0:443
+        mode tcp
+        balance roundrobin
+        server <cluster-name>-worker-0 192.168.3.18:443 check
+        server <cluster-name>-worker-1 192.168.2.228:443 check
+        server <cluster-name>-worker-2 192.168.1.253:443 check
+listen <cluster-name>-apps-80
+        bind 0.0.0.0:80
+        mode tcp
+        balance roundrobin
+        server <cluster-name>-worker-0 192.168.3.18:80 check
+        server <cluster-name>-worker-1 192.168.2.228:80 check
+        server <cluster-name>-worker-2 192.168.1.253:80 check
 ```
 
-You could also specify a specific IP address with /32 if you wish.
+#### DNS Lookups
 
-You can verify the operation of the load balancer now if you wish, using the curl commands given below.
-
-Now the DNS entry for `api.<cluster name>.<base domain>` needs to be updated to point to the new load balancer:
+To ensure that your API and apps are accessible through your load balancer, [create or update DNS entries](#Create API and Ingress DNS Records) for them. To use your new load balancing service for external traffic, make sure the IP address for these DNS entries is the IP address your load balancer is reachable at.
```

Comment:
The anchor should be #create-api-and-ingress-dns-records, otherwise the link doesn't render. The sentence ends bizarrely too.
```suggestion
To ensure that your API and apps are accessible through your load balancer, [create or update DNS entries](#create-api-and-ingress-dns-records) for them. To use your new load balancing service for external traffic, make sure the IP address for these DNS entries is the IP address your load balancer:
```